modules/serverless-workflow-autoscaling.adoc

```diff
@@ -0,0 +1,28 @@
+// Module is included in the following assemblies:
+//
+// serverless/knative-client.adoc
+
+[id="autoscaling-workflow_{context}"]
+= Autoscaling workflow using Knative Client
+
+You can access autoscaling capabilities by using `kn` to modify Knative services without editing YAML files directly.
+
+Use the `service create` and `service update` command with the appropriate flags to configure the autoscaling behavior.
+
+[cols="1,3",options="header"]
+|====
+| Flag 
+| Description
+
+| `--concurrency-limit int`       
+| Hard limit of concurrent requests to be processed by a single replica.
+
+| `--concurrency-target int`      
+| Recommendation for when to scale up based on the concurrent number of incoming requests. Defaults to `--concurrency-limit`.
+
+| `--max-scale int`               
+| Maximum number of replicas.
+
+| `--min-scale int`             
+| Minimum number of replicas.
+ 
```

Comment:
You're missing the |==== to close the table, so when you look at the rest of the assembly rendered, it's all funky now inside this table.